Misc changes: Add more XCM SDK examples, add Web3.js common errors, and a few quick fixes (#812)
moonbeam-foundation/moonbeam-docs: tokens/governance/voting.md

Reviewer: Typo on Line 69 - recommend should be recommended

Reviewer: Line 97 typo, should be diligence

Reviewer: Line 105 typo should be referendum

Reviewer: Line 141 beginning of sentence not capitalized

Reviewer: Line 145 2nd instance misspelled should be continuously

Reviewer: Line 147 should be enactment

Reviewer: Line 151 chose should be chosen

## Introduction {: #introduction }

Referenda are simple, inclusive, and stake-based voting schemes. Each referendum has a proposal associated with it that suggests an action to take place. In OpenGov, each referendum will have a specified Origin class that the proposal will be executed with, and each Origin has its own Track that proposals will process through. Although referenda are completed by a common process, the requirements for approval are Track-specific.

Token holders can vote on referenda using their own tokens, including those that are locked in staking. The weight of a vote is defined by two factors: the number of tokens locked and the lock duration (called Conviction). This is to ensure that there is an economic buy-in to prevent vote-selling. Consequently, the longer you are willing to lock your tokens, the stronger your vote will be weighted. You also have the option of not locking tokens at all, but the vote weight is drastically reduced.

> In Moonbeam, users will be able to create and vote on proposals using their H160 address and private key, that is, their regular Ethereum account!

Reviewer: ```suggestion
In Moonbeam, users are able to create and vote on proposals using their H160 address and private key, that is, their regular Ethereum account!
```
